src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -42,9 +42,7 @@ const SubscriberHistory = ({ location }) => {
           </div>
         </header>
       )}
-      content={({ subscriber }) => (
-        <SubscriberSentCampaigns subscriber={subscriber} location={location} />
-      )}
+      content={({ subscriber }) => <SubscriberSentCampaigns subscriber={subscriber} />}
```

Comment:
Now that we do not need to inject location this way should work, right?

```suggestion
      content={<SubscriberSentCampaigns />}
```

Reply:
It could work, but I have to get sent campaigns using the subscriber email in the url instead in subscriber object. I have to make this change.

Reply:
We have already discussed and fixed it.